src/unitxt/operators.py

```diff
@@ -1985,6 +1986,80 @@ def process(self, stream: Stream, stream_name: Optional[str] = None) -> Generato
                 yield instance
 
 
+class MinimumOneExamplePerLabelRefiner(StreamRefiner):
+    """A class used to include at least one example per label.
+
+    For each instance, a signature is constructed from the values of the instance in specified input 'fields'.
+    By discarding instances from the input stream, MinimumOneExamplePerLabelRefiner maintains at least one
+    example per label. MinimumOneExamplePerLabelRefiner shuffles the results to avoid having one element
+    from each class first and then the rest . If max instance is not set, the original stream will be used
```

Comment:
Need to mention the resulting stream is shuffled.

Reply:
more than what I already wrote? 
``` 
...MinimumOneExamplePerLabelRefiner shuffles the results to avoid having one element from each class first and then the rest
```